runtime/experimental/export.Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM bref/runtime/php:latest
+# Use the bash shell, instead of /bin/sh
+SHELL ["/bin/bash", "-c"]
+
+# We do not support running pear functions in Lambda
+RUN rm -rf /opt/bref/lib/php/PEAR
+RUN rm -rf /opt/bref/share/doc
+RUN rm -rf /opt/bref/share/man
+RUN rm -rf /opt/bref/share/gtk-doc
+RUN rm -rf /opt/bref/include
+RUN rm -rf /opt/bref/tests
+RUN rm -rf /opt/bref/doc
+RUN rm -rf /opt/bref/docs
+RUN rm -rf /opt/bref/man
+RUN rm -rf /opt/bref/www
+RUN rm -rf /opt/bref/cfg
+RUN rm -rf /opt/bref/libexec
+RUN rm -rf /opt/bref/var
+RUN rm -rf /opt/bref/data
+
+WORKDIR /opt
+RUN LD_LIBRARY_PATH= yum -y install zip
+RUN export PHP_ZIP_NAME=/export/php-$(php -r '$version = explode(".", phpversion());printf("%d.%d", $version[0], $version[1]);'); \
+ mkdir -p /export; \
+ zip -y -o -9 -r ${PHP_ZIP_NAME}.zip . -x "*php-cgi"; \
+ cp ${PHP_ZIP_NAME}.zip ${PHP_ZIP_NAME}-cli.zip; \
+ zip -d ${PHP_ZIP_NAME}-cli.zip bref/sbin/php-fpm bin/php-fpm; \
+ cp ${PHP_ZIP_NAME}.zip ${PHP_ZIP_NAME}.fpm.zip; \
+ zip -d ${PHP_ZIP_NAME}.fpm.zip bref/bin/php /bin/php
```

Comment:
The files exported are:

- `php-7.2.zip` -> ✅ PHP CLI layer
- `php-7.2.fpm.zip` -> ✅ FPM layer
- `php-7.2-cli.zip` -> is it a duplicate of the first one?

Reply:
**php-7.2.zip** - This contains everything we could possibly build.
**php-7.2.fpm.zip** - Copy of  **php-7.2.zip**, removes the _CLI SAPI_, only provides the _FPM SAPI_.
**php-72-cli.zip** - Copy of  **php-7.2.zip**, removes the _FPM SAPI_, only provides the _CLI SAPI_.